pythagorean_thm.py

```diff
@@ -0,0 +1,27 @@
+'''
+This script will help solve the pythagorean theorem.
+'''
+
+import math
```

Comment:
This function would be useful for a lot of purposes in my workflow (not just for a = 3 and c = 5), can we add the ability for user inputs with argparse?